python/pyspark/ml/tuning.py

```diff
@@ -431,7 +433,9 @@ def copy(self, extra=None):
         """
         if extra is None:
             extra = dict()
-        return TrainValidationSplitModel(self.bestModel.copy(extra))
+        bestModel = self.bestModel.copy(extra)
+        validationMetrics = self.validationMetrics
```

Comment:
It might be good to explicitly also copy the validation metrics rather than just the reference?